### DISCUSSION TITLE
Module '"@badeball/cypress-cucumber-preprocessor"' has no exported member 'AfterAll'

### DISCUSSION THREAD

**Discussion by kgalushchak** [badeball/cypress-cucumber-preprocessor#1119] [Q&A] [answered]
Hello! I am want to use `AfterAll` hook in my test as per https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/features/hooks_ordering.feature.
I use Cypress with Cucumber and Typescript.
When trying to `import { AfterAll } from '@badeball/cypress-cucumber-preprocessor'` I see an error:
`Module '"@badeball/cypress-cucumber-preprocessor"' has no exported member 'AfterAll'`

![Screenshot 2023-11-14 at 16 24 28](https://github.com/badeball/cypress-cucumber-preprocessor/assets/82284603/2461881a-7b50-40f6-b27a-7a507eb381a0)

```
   "@badeball/cypress-cucumber-preprocessor": "^18.0.2",
   "cypress": "^12.17.3",
```

What am I doing wrong?

**Comment by badeball** [comment, marked as answer]
This was released in v19.1.0, as per changelog: https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CHANGELOG.md#v1910
